src/main/java/core/basesyntax/PictureOfFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface PictureOfFigure {
+    String getDraw();
```

Comment:
```suggestion
public interface InformationDrawer {
    String draw();
```

as we can use this interface not only for figures